app/src/main/java/agency/five/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,51 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun UserScoreProgressBar(
+    modifier: Modifier = Modifier,
+    userScore: Float,                   //0.0 - 10.0
```

Comment:
Instead of using comments you can define FloatRange like this: 
@FloatRange(from = 0.0, to = 10.0) userScore: Float.
Also not in scope of this homework, but optional parameters (the ones with default) should always go after necessary ones, this way you can call the function without explicit parameter names (when you don't pass optional params). So modifier should go after userScore, also modifier should always be the first optional parameter.